hopsworks-web/yo/Gruntfile.js

```diff
@@ -94,7 +94,7 @@ module.exports = function (grunt) {
           livereload: '<%= connect.options.livereload %>'
         },
         files: [
-          '<%= yeoman.app %>/{,*/}*.html',
+          '<%= yeoman.app %>/{,**/}*.html',
```

Comment:
@robzor92 what does this fix?

Reply:
it fixes being able to use images from our images folder in hopsworks inside yo/app/views/somefolder, whereas currently you can only do it from yo/app/views. I don't need it anymore but figure it can be useful. @ErmiasG helped me with it.

Reply:
thanks